### DISCUSSION TITLE
Can I have a live view of more than one table at once?

### DISCUSSION THREAD

**Discussion by pitosalas** [Textualize/rich#3140] [Q&A]
I need to have three tables which are all updated "live". How can I do that? It seems to me like:

```
with Live(generate_table(), refresh_per_second=4) as live:
    for _ in range(40):
        time.sleep(0.4)
        live.update(generate_table())
```
Allows only for one table to be genrated. I looked at the Tree class but that doesn't seem to do the trick. Can someone help me understand please?

**Comment by hemna** [comment]
I have a similar need.  I want to create a layout and have a few tables in that layout.  each of those tables would be live and updatable.  It's unclear how to do this.

**Reply by willmcgugan** [reply]
Everything in Rich is composable. You can put tables inside tables.
